tests/testthat/test-profile_emissions.R

```diff
@@ -467,19 +397,14 @@ test_that("with no match preserves unmatched products (#193)", {
   ecoinvent_europages <- read_csv(toy_ecoinvent_europages())
   isic_name <- read_csv(toy_isic_name())
 
-  out <- profile_emissions(
+  expect_error(profile_emissions(
     companies,
     co2,
     europages_companies,
     ecoinvent_activities,
     ecoinvent_europages,
     isic_name
-  )
-
-  expect_equal(out$companies_id, unique(companies$companies_id))
-
-  product <- unnest_product(out)
-  expect_true("unmatched" %in% product$activity_uuid_product_uuid)
+  ), class = "all_benchmark_is_na")
```

Comment:
@maurolepore May you please give the reference to the class `all_benchmark_is_na`?

Reply:
Here it is https://github.com/2DegreesInvesting/tiltIndicatorAfter/pull/214#discussion_r1589416217

Reply:
I meant, what's the origin of the class `all_benchmark_is_na`? I can't see any class in `tiltIndicator` with the same name.